src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -348,6 +388,37 @@ private void EnsureSignature()
             }
         }
 
+        private void EnsureAccessorsBinding()
+        {
+            if (!_state.HasComplete(CompletionPart.FinishPropertyAccessorsBinding))
+            {
+                if (_state.NotePartComplete(CompletionPart.StartPropertyAccessorsBinding))
+                {
+                    // Ensure the binding is done so we guarantee that we have the BackingField set if necessary.
+                    // If we already have a backing field. We should still do binding since we create a backing field just if
+                    // we see an initializer. In this case binding is needed to know whether it's an auto property.
+                    if (this is SourcePropertySymbol { ContainsFieldKeyword: true } propertySymbol)
```

Comment:
>this is SourcePropertySymbol { ContainsFieldKeyword: true } propertySymbol [](http://example.com/codeflow?start=24&length=74)

Is it possible that we already created the backing field?

Reply:
@AlekseyTs Yes we could already have created the backing field. I'm explicitly stating this in the comment. This is very related to https://github.com/dotnet/roslyn/pull/57076#discussion_r767318435.

Reply:
>>Is it possible that we already created the backing field?

>Yes we could already have created the backing field. I'm explicitly stating this in the comment. This is very related to [https://github.com/dotnet/roslyn/pull/57076#discussion_r767318435](https://github.com/dotnet/roslyn/pull/57076#discussion_r767318435).

Then why we are still binding accessors for a scenario when the field has been created already?

Reply:
@AlekseyTs This is to ensure that IsAutoProperty flag is set. If the field was created, this doesn't mean IsAutoProperty flag is set.